compiler/passes/ResolveScope.cpp

```diff
@@ -980,11 +980,11 @@ Symbol* ResolveScope::lookupPublicImports(const char* name) const {
   Symbol *retval = NULL;
```

Comment:
Can't comment on that specific line: I think this function should be renamed to `lookupPublicVisStmts` or `lookupPublicUsesAndImports`